packages/web/src/pages/chat-page/components/ChatMessageListItem.module.css

```diff
@@ -50,8 +50,8 @@
 
 .textWrapper {
   background-color: var(--bubble-color);
-  padding: var(--harmony-unit-4);
   text-align: left;
+  max-width: 448px;
```

Comment:
is the max width of the chat bubble 448px? just wondering where the number is from